bin/container-setup

```diff
@@ -61,6 +61,15 @@ elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
 else
 
   echo "Oof! Sorry! This OS is unsupported! :("
+  echo "If your are on Windows OS and have Docker or Podman for windows installed:"
```

Comment:
Typo: it should be "If you're" or "If you are"

Reply:
Okay 👍🏾